Assets/Scripts/Enemy.cs

```diff
@@ -29,7 +29,16 @@ public class Enemy : MonoBehaviour
     [SerializeField] private bool isHolding = false;
     [SerializeField] private bool isCharging = false;
     private int chargeDirection; //1 = up, 2 = down, 3 = right, 4 = left
-    [SerializeField] private bool bomber = false;
+    [SerializeField] private bool boss = false;
+    [SerializeField] private int pattern;
+    [SerializeField] private float timeToNextPattern = .5f;
+    private float patternDucation = 2.5f;
+    [SerializeField] private GameObject bomb;
+    [SerializeField] private GameObject electricField;
+    public bool isDamaged = false;
+    [SerializeField] private float invincibleTime;
+    private float toBeVincible;
+    public static int bossHp;

```

Comment:
a helpful organization tip is to always list your variables/fields in a specific order in every script. I like to put mine in order from most exposed to least exposed top to bottom. so, I would put public properties and variables at the top, serielized fields next, and private variables last. it helpful to organize them together so things are easier to find and it helps other people read your code too